feat(RN): Troubleshoot sensitive transaction data (#8767)
getsentry/sentry-docs: src/platforms/react-native/troubleshooting/index.mdx

  ],
});
```

> ## Transaction Data Contains Sensitve Information

Reviewer: ```suggestion
## Routing Transaction Data Contains Sensitve Information
```

> ## Transaction Data Contains Sensitve Information

Reviewer 2: ```suggestion
## Transaction Data Contains Sensitive Information
```

> The SDK might send sensitive data such as username and password for auto-generated transactions before 5.15.2 from navigation routing parameters. You can fix this issue by either updating to [5.15.2](https://github.com/getsentry/sentry-react-native/releases/tag/5.15.2) or higher or dropping the data manually with `beforeSendTransaction`:

Reviewer: ```suggestion
If you use automatic routing instrumentation for React Navigation, sensitive data you put in routing parameters can get sent to Sentry. You can prevent this by either updating the SDK to [5.15.2](https://github.com/getsentry/sentry-react-native/releases/tag/5.15.2) or higher, or dropping the data manually with `beforeSendTransaction`:
```
